src/app/health/health-event-dialog.component.ts

```diff
@@ -46,4 +49,21 @@ export class HealthEventDialogComponent implements OnInit {
     this.router.navigate([ 'event', { id: this.data.user, eventId: event._id } ], { relativeTo: this.data.route });
   }
 
+  countdown() {
+    this.timer();
+    const intervalId = setInterval(() => {
+      this.timer(intervalId);
+    }, 1000);
+  }
+
+  timer(intervalId?) {
+    const time = (10000 - (new Date(Date.now()).getTime() - new Date(this.event.date).getTime())) / 1000;
```

Comment:
@jsschf This is off.  The number at the start should be 300000 (5 minutes in ms).

Reply:
It would be best to set this as a property and replace the 300000 in the constructor as well.  That way it'll be easy to change the edit time limit.